README.md

```diff
@@ -152,7 +152,11 @@ See the full [tutorial](docs/tutorial.md) for more details.
   build/sbt generate
   ``` 
   This will regenerate the OpenAPI data models in the UC server and data models + APIs in the client SDK.
-
+- To create a tarball that can be used to deploy the UC server or run the CLI, run the following:
+  ```sh
+  build/sbt createTarball
+  ```
+  This will create a tarball in the `target` directory.
```

Comment:
Add a separate section above Compiling and Testing and explain how to deploy using the tarball